cosmic_ray/operators/remove_decorator.py

```diff
@@ -10,7 +10,7 @@ class RemoveDecorator(Operator):
 
     def visit_FunctionDef(self, node):  # noqa
         decorator_candidates = [x for x in node.decorator_list
-                                if x.id not in self.REGULAR_DECORATORS]
+                                if hasattr(x, 'id') and x.id not in self.REGULAR_DECORATORS]
```

Comment:
Actually, shouldn't this be "if not hasattr(x, 'id') or x.id not in self.REGULAR_DECORATORS"? That is, we want to remove decorators which aren't in "REGULAR_DECORATORS", and this include any decorator without an "id" field.